infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -47,6 +53,9 @@ public class CorfuServer {
     private static volatile boolean shutdownServer = false;
     // If set to true - triggers a reset of the server by wiping off all the data.
     private static volatile boolean cleanupServer = false;
+    @Getter
+    @Setter
+    private static boolean keepServerComponents = false;
```

Comment:
Can you rename it to something like - reloadable?

Reply:
Done, thank you!